### ISSUE TITLE
demo有中文，但是paper里提的训练集都是英文的

### ISSUE THREAD

**Issue by lszhou0126** [jixiaozhong/Sonic#1]
想问下中文效果怎么实现的，像hallo这些用英文数据集，中文demo好像不太理想

**Comment by jixiaozhong** [comment]
音频特征对与多语种泛化性很重要，开源数据集中也包含少量多语种数据

**Comment by jixiaozhong** [comment]
We appreciate your interest in our work.

Primarily, the support for Chinese is heavily reliant on the diversity of the dataset. We utilized a total of 60,000 clips from VFHQ, VoxCeleb2, and CelebVText, which include a small amount of clips from multiple languages, in addition to English. This provides us with a significant advantage in terms of data scale compared to Hallo's 2,000 clips. 

In addition, the audio features pre-trained on multilingual corpora inherently align different languages, which has a certain generalization. We employed the Whisper-Tiny model, trained on multiple languages, while Hallo used Wav2Vec, which may have only been trained on basic English data. We incorporated temporal audio learning into our model, which provides a larger perceptual range for audio features and enhances the model's understanding of these features. Compared to Hallo, we achieved superior results in lip synchronization.
